src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -44,13 +47,19 @@
                                 data_client_path.format('KeyVaultClient.get_keys'))
 cli_keyvault_data_plane_command('keyvault key list-versions',
                                 data_client_path.format('KeyVaultClient.get_key_versions'))
+cli_keyvault_data_plane_command('keyvault key list-deleted',
```

Comment:
list and list-deleted have some conflicting arguments in some cases and return different types in all, this is why I created them as separate commands